dist/tools/openocd/openocd.sh

```diff
@@ -159,6 +187,15 @@ do_flash() {
             exit $RETVAL
         fi
     fi
+
+    # In case of binary file, IMAGE_OFFSET should include the flash base address
+    # This allows flashing normal binary files without env configuration
+    if _is_binfile ${IMAGE_FILE}; then
+        # hardwritten to use the first bank
+        FLASH_ADDR=$(_flash_address 1)
```

Comment:
What do you call bank here?

Reply:
I used the term from openocd documentation: http://www.openocd.org/doc/html/Flash-Commands.html and here http://repo.or.cz/openocd.git/blob_plain/HEAD:/tcl/target/stm32f1x.cfg (`flash bank`).

From what I understood, it is a contiguous area of ROM.

By setting a `1` here, I assume that, if the board has multiple memory banks, the first one is used.